cookbooks/bcpc-hadoop/recipes/graphite_to_zabbix.rb

```diff
@@ -229,10 +229,16 @@
         end # End of "if (trigger_id = existing_triggers[trigger_name]).nil?"
 
         # Create/Update Actions
+        if node[:bcpc][:hadoop][:zabbix][:enable_alarming] == false
```

Comment:
Just a nit but to point it out, you could restructure this to avoid the `== false` and do the expected instead of the negation first; e.g.:
````
if node[:bcpc][:hadoop][:zabbix][:enable_alarming]
  action_status = status
else
  action_status = 1
end
````
Or more concise:
`action_status = node[:bcpc][:hadoop][:zabbix][:enable_alarming] ? status : 1`